dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/datavalue/DefaultDataValueService.java

```diff
@@ -53,7 +53,7 @@
 /**
  * Data value service implementation. Note that data values are softly deleted,
  * which implies having the deleted property set to true and updated.
- * 
+ *

```

Comment:
@samaxes not sure why these files were changed? looks like white space?

Reply:
@jason-p-pickering there's a new method called `updateDataValues`. Formatting (and white space) has changed because we format classes we touch with the DHIS2 conventions we imported into IDEA.